ReactCommon/jsi/JSCRuntime.cpp

```diff
@@ -928,7 +928,7 @@ jsi::WeakObject JSCRuntime::createWeakObject(const jsi::Object& obj) {
 #endif
 }
 
-jsi::Value JSCRuntime::lockWeakObject(const jsi::WeakObject& obj) {
+jsi::Value JSCRuntime::lockWeakObject(__unused const jsi::WeakObject& obj) {
```

Comment:
@hramos Maybe we need to postpone to fix this warning, because `__unused` is the macro provided by Apple clang, but `JSCRuntime` is the common component run on all platforms, please see https://github.com/facebook/react-native/pull/23705, we discussed the details at there.

Reply:
Thanks for bringing that up! In light of that, I'll go ahead and close this PR.